src/gather_classes.rs

```diff
@@ -273,7 +274,13 @@ fn update_report(
 fn get_calls<'a>(obj: &'a Value) -> Box<dyn Iterator<Item = &Value> + 'a> {
     // TODO (#72) Put debug logging here under a core::option_env flag so it is normally hidden.
     match obj {
-        Value::String(_same) => Box::new(::std::iter::empty()),
+        Value::String(string) => {
+            if string.starts_with(SAME) {
```

Comment:
There is a function were we wrap this small logic. Can you use it here? That way when there is always a different way to compare for SAME we just need to update that single part of the code